e2e/opts.go

```diff
@@ -8,4 +8,5 @@ type scenarioRunOpts struct {
 	clusterConfig *Cluster
 	scenario      *Scenario
 	nbc           *datamodel.NodeBootstrappingConfiguration
+	scriptless    bool
```

Comment:
super duper nit: this feels somewhat incomplete to me, `scriptlessVHD` maybe makes more sense to properly highlight what the scenario option is.

taking the cue from clusterConfig instead of just Config. feel free to ignore, just my opinion.

Reply:
lol `scriptless` was my original suggestion, I think `scriptlessVHD` is fine too

Reply:
i think i prefer simplicity of `scriptless` and we apparently we might rebrand again XD so just going to keep as is